src/App/Game/GameOver.tsx

```diff
@@ -35,6 +35,18 @@ export class GameOver extends React.Component<IGameOverProps, {}> {
     } else {
       playAgainText = 'Do you want to play again? Check out our games below.';
     }
+    if (typeof this.props.gameArgs !== 'undefined') {
+      ReactGA.event({
+        category: 'GameOver',
+        label: this.props.gameArgs.gameCode,
```

Comment:
Dont have this if. You can assume every GameOver should have a gameCode. If some tests dont have it, change the tests, do NOT add a branch in the code just because it is easier to test, specially because that state is invalid in normal conditions (not during test).

Reply:
> do NOT add a branch in the code just because it is easier to test

![DEEBC9B1-DF3A-413B-B767-C46609CCD45D](https://user-images.githubusercontent.com/304383/55413407-4292f400-5537-11e9-90a2-0b4077775d99.jpeg)


You are right.  Fixed :)